### PR TITLE
toaplan2.cpp: Add Battle Garegga location test

### PR DESCRIPTION
**Pull request by Mikejmoffitt**
* Add bgareggap loctest set to toaplan2.cpp
* Fix YM2151/OKI level balancing for Battle Garegga and Armed Police Batrider

### CODE REVIEW

File: src/mame/drivers/toaplan2.cpp [hunk 1]
```diff
@@ -5691,6 +5740,7 @@ GAME( 1994, kingdmgp,    0,        shippumd,   kingdmgp,   toaplan2_state, empty
 GAME( 1994, shippumd,    kingdmgp, shippumd,   shippumd,   toaplan2_state, empty_init,      ROT270, "Raizing / Eighting", "Shippu Mahou Daisakusen (Japan)", MACHINE_SUPPORTS_SAVE )
 
 GAME( 1996, bgaregga,    0,        bgaregga,   bgaregga,   toaplan2_state, init_bgaregga,   ROT270, "Raizing / Eighting", "Battle Garegga (Europe / USA / Japan / Asia) (Sat Feb 3 1996)", MACHINE_SUPPORTS_SAVE )
+GAME( 1996, bgareggap,   bgaregga, bgaregga,   bgaregga,   toaplan2_state, init_bgaregga,   ROT270, "Raizing / Eighting", "Battle Garegga (Prototype) (Wed Jan 17 1996)", MACHINE_SUPPORTS_SAVE )
```
Comment (cuavas) [inline src/mame/drivers/toaplan2.cpp:5743]: This isn’t a prototype, it’s a location test version.  Also, anything that isn’t actually part of the title should only capitalise proper nouns.

File: src/mame/mame.lst [hunk 2]
```diff
@@ -40177,6 +40177,7 @@ bbakraid                        // (c) 1999 8ing
 bbakraidc                       // (c) 1999 8ing
 bbakraidj                       // (c) 1999 8ing
 bbakraidja                      // (c) 1999 8ing
+bgareggap                       // (c) 1996 Raizing/8ing
```
Comment (cuavas) [inline src/mame/mame.lst:40180]: These should be kept sorted.

**Review by cuavas — CHANGES_REQUESTED** [review]
Please follow convention for titles, and edit your pull request to give it a better name.

**Comment by Mikejmoffitt** [comment]
> Please follow convention for titles, and edit your pull request to give it a better name.

I don't know how github squashed the first line I wrote in the commit.

File: src/mame/drivers/toaplan2.cpp (hunk 1 above)
Comment (Mikejmoffitt) [reply src/mame/drivers/toaplan2.cpp]: Understood; there are examples in this file that violate that convention e.g. "Pips & Bibis / Whoopee!!  (Prototype)" or "Ghox (Spinner with .."

File: src/mame/mame.lst (hunk 2 above)
Comment (Mikejmoffitt) [reply src/mame/mame.lst]: Resolved in latest commit.

**Pull request merged**